PirateLexer/Tokens/KeyWordService.cs

```diff
@@ -9,71 +9,71 @@ public class KeyWordService : IKeyWordService
 
     private string[] controlKeywords = new string[] { "if", "else", "for", "to", "foreach", "in", "while", "func", "class", "new", "return" };
 
-    public TokenTypeKeyword GetTypeKeyword(string idString)
+    public TokenType GetTypeKeyword(string idString)
```

Comment:
Change to C11 switch expression

Reply:
example:
    public static Orientation ToOrientation(Direction direction) => direction switch
    {
        Direction.Up    => Orientation.North,
        Direction.Right => Orientation.East,
        Direction.Down  => Orientation.South,
        Direction.Left  => Orientation.West,
        _ => throw new ArgumentOutOfRangeException(nameof(direction), $"Not expected direction value: {direction}"),
    };